### PR TITLE
Making README shorter

### PR DESCRIPTION
**Pull request by jloleysens**
* Move Roadmap to just above contributing so that Getting started is on top
* Added link to releases page
* Tightened up language _a lot_

Preview: https://github.com/animo/aries-cli/tree/Making-README-shorter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,35 +61,43 @@ echo "Coming soon!"
 Write-Output "Coming soon!"
 ```
 
+### Binaries
+
+See [binaries](https://github.com/animo/aries-cli/releases).
+
```
Comment (anagoessens) [inline README.md:67]:
```suggestion
### Git install

```sh
cargo install --git https://github.com/animo/aries-cli
```
```

Comment (blu3beri) [reply README.md:67]: I wouldn't say git install, but `Install via Cargo` or `Cargo install`

Comment (anagoessens) [reply README.md:67]: Adjusted, but honestly suggest removing it / moving it to the advanced installation as soon as the rest of the install methods are updated.

File: README.md [hunk 2]
```diff
@@ -77,35 +61,43 @@ echo "Coming soon!"
...
+This command will set up the community agent. To set up your own agent edit
+the configuration file by adding your agent URL.
 
-If you are getting started with the tool, we recommend enabling informational logs by
-passing the `--verbose` (or `-v`) flag to get more information about what the CLI is
-doing. We also support stacking verbosity up to three levels for debug logs: `-vv` and `-vvv`.
+If you are getting started with the tool we recommend enabling informational logs by
+passing the `--verbose` (or `-v`).
 
 ## Common examples
```
[the remark this answers is not in the document]
Comment (jloleysens) [reply README.md:84]: i think it is kind of a natural flow for new users after getting set up which I think is nice. Ideally we'd send them to the docs that'll be more comprehensive after that. In short, I think it'd be nice to keep for now, then possibly link out to docs and shorten or remove in future. WDYT?

**Comment by jloleysens** [comment]
I'm going to go ahead and merge this for now @blu3beri @anagoessens happy to take follow up requests for changes if you feel strongly about it!

**Pull request merged**